Docs/installation/debian_jessie.md

```diff
@@ -0,0 +1,54 @@
+# Kalliope installation on Debian Jessie
+
+## Automated install
+
+Clone the project
+```
+cd
+git clone https://github.com/kalliope-project/kalliope.git
+```
+
+Run the install script.
+```
+./kalliope/install/install_kalliope.sh
+```
+
+## Manual install
+
+Edit `/etc/apt/sources.list` and check that your mirror accept "non-free" package
+```
+deb http://ftp.fr.debian.org/debian/ jessie main contrib non-free
+deb-src http://ftp.fr.debian.org/debian/ jessie main contrib non-free
```

Comment:
This section should be moved directly after the git clone section.
Because it is also used in the install script